### ISSUE TITLE
%opts break after error

### ISSUE THREAD

**Issue by basnijholt** [ioam/holoviews#1463]
Reproduce with
```python
import holoviews as hv
hv.notebook_extension()
%opts Image [cmap='viridis']
```

then try:
```python
%opts Image (cmap='viridis')
```
which will print:
```
Unexpected plot option 'cmap' for Image in loaded backend 'matplotlib'.

Possible keywords in the currently active 'matplotlib' backend are: ['apply_extents', 'apply_ranges', 'apply_ticks', 'aspect', 'bgcolor', 'cbar_padding', 'cbar_ticks', 'cbar_width', 'clipping_colors', 'colorbar', 'fig_alpha', 'fig_bounds', 'fig_inches', 'fig_latex', 'fig_rcparams', 'fig_size', 'final_hooks', 'finalize_hooks', 'fontsize', 'initial_hooks', 'invert_axes', 'invert_xaxis', 'invert_yaxis', 'invert_zaxis', 'labelled', 'logx', 'logy', 'logz', 'normalize', 'projection', 'show_frame', 'show_grid', 'show_legend', 'show_title', 'situate_axes', 'sublabel_format', 'sublabel_position', 'sublabel_size', 'symmetric', 'title_format', 'xaxis', 'xrotation', 'xticks', 'yaxis', 'yrotation', 'yticks', 'zaxis', 'zrotation', 'zticks']

If you believe this keyword is correct, please make sure the backend has been imported or loaded with the notebook_extension.
In [ ]:

```

**Comment by jlstevens** [comment]
Thanks for reporting this.

It now seems to think the style option is a plot option for some reason. I'll need to investigate...

**Comment by jlstevens** [comment]
Should be fixed in #1465. 

Thankfully it was an easy fix of the options system for a change!
